src/Windows/Avalonia.Win32/Avalonia.Win32.csproj

```diff
@@ -5,8 +5,15 @@
     <!-- We still keep BinaryFormatter for WinForms compatibility. -->
     <EnableUnsafeBinaryFormatterSerialization>true</EnableUnsafeBinaryFormatterSerialization>
   </PropertyGroup>
-  <ItemGroup>
+  <ItemGroup Condition="!$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net6.0'))">
     <PackageReference Include="System.Numerics.Vectors" Version="4.5.0" />
+    <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="6.0.0" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Windows.CsWin32" Version="0.3.106">
```

Comment:
I also want to use this PR as a first attempt to introduce CsWin32 into the repository.
If this source generator will get annoying enough, we can rewrite it to old-style interop.
I would want to keep it though, if possible.

Reply:
I want to use this for the spell checking systems as well. So this is quite helpful to introduce it now.